### PR TITLE
feat: implement sealed secrets for redis

### PR DESCRIPTION
**Pull request by ambaliyav**

### CODE REVIEW

File: charts/redis/values.yaml [hunk 1]
```diff
@@ -9,3 +9,8 @@ redis:
     port: 6379
   db_name: 1
   expiry: 3600
```
Comment (krishnamadhavan) [inline charts/redis/values.yaml]: We do not need to indent this inside the redis block. Let's keep it outside.
https://github.com/WHOAcademy/helm-charts/pull/10/files#diff-374504b976e140fee1dc6d6f14eab2bce487f9791c82c86d3b8e474dd55ff9e5R2-R11

File: charts/redis/values.yaml [hunk 2]
```diff
@@ -9,3 +9,8 @@ redis:
...
+
+sealed_secrets:
+  enabled: false
+
+application_name:
```
Comment (krishnamadhavan) [inline charts/redis/values.yaml]: Can we put this inside the sealed_secrets ? And can we rename it as `sealed_secret_name` ?

File: charts/redis/templates/Secret.yaml [hunk 3]
```diff
@@ -1,3 +1,5 @@
+{{- if not .Values.sealed_secrets.enabled }}
+{{- if .Values.managed_redis.disabled }}
```
Comment (krishnamadhavan) [inline charts/redis/templates/Secret.yaml]: We don't need this line as from our application we won't call this chart if we use managed_redis.

File: charts/redis/values.yaml (hunk 1 above)
Comment (ambaliyav) [reply charts/redis/values.yaml]: It's updated

File: charts/redis/values.yaml (hunk 2 above)
Comment (ambaliyav) [reply charts/redis/values.yaml]: Done

File: charts/redis/templates/Secret.yaml (hunk 3 above)
Comment (ambaliyav) [reply charts/redis/templates/Secret.yaml]: It's removed

**Comment by krishnamadhavan** [comment]
@ambaliyav  Can we bump the chart version ?

**Review by krishnamadhavan — APPROVED** [review]

**Pull request merged**